cmd/build.go

```diff
@@ -37,7 +37,7 @@ func gatherContext(rootDir string, w io.Writer, archiveType string) error {
 	} else {
 		gi, err = ignore.CompileIgnoreFileAndLines(giPath, lines...)
 		if err != nil {
-			ui.Errorf("Error compiling .gitignore file:", err)
+			ui.Errorf("Error compiling .gitignore file %s:", err)
```

Comment:
This is incorrect formatting. `go run .` doesn't care but Go test cases won't compile if the formatting function isn't correct.